app/models/feedback_response_map.rb

```
@@ -20,4 +20,53 @@ def get_title
+      all_review_response_ids = []
+
+      temp_responses.each do |response|
+        next if seen_map_round_keys.include?(response.map_id)
+
+        seen_map_round_keys << response.map_id
+        all_review_response_ids << response.id
+      end
+
+      [authors, all_review_response_ids]
+    end
+  end
```

RuboCop (1.86.1)

[convention] Assignment Branch Condition size for `feedback_response_report` is too high. [<15, 34, 14> 39.71/20]

(Metrics/AbcSize)